<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusBarWidget</class>
 <widget class="QWidget" name="StatusBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>23</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,0,10,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="nodeCountLabel">
     <property name="text">
      <string>Node count:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="lcdNodeCount">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip">
      <string>This displays the count of story nodes</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="storyNameLabel">
     <property name="text">
      <string>Story name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="storyNameLineEdit">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Here you can edit the name of the story</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="graphicFileLabel">
     <property name="text">
      <string>Display data:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="graphicFileIndicator">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
